angular/angular8/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
+
+if (environment.production) {
+  enableProdMode();
```

Comment:
It doesn't hurt to leave this here, though we will never hit this condition because we are going to be working on this problem live with the candidates, so it'll always be in dev mode since we're going to be hot reloading changes. I'm okay if we omit this specific setting check for now.

Reply:
Removed